Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean

```diff
@@ -159,21 +159,27 @@ theorem card_mul_mul_card_le_card_mul_mul_card_mul (A B C : Finset α) :
 #align finset.card_mul_mul_card_le_card_mul_mul_card_mul Finset.card_mul_mul_card_le_card_mul_mul_card_mul
 #align finset.card_add_mul_card_le_card_add_mul_card_add Finset.card_add_mul_card_le_card_add_mul_card_add
 
-/-- **Ruzsa's triangle inequality**. Add-sub-sub version. -/
+/-- **Ruzsa's triangle inequality**. Mul-div-div version. -/
+@[to_additive card_add_mul_le_card_sub_mul_card_sub
+      "**Ruzsa's triangle inequality**. Add-sub-sub version."]
```

Comment:
The style is rather to not indent the additive docstring
```suggestion
"**Ruzsa's triangle inequality**. Add-sub-sub version."]
```

Reply:
It's indented in the rest of the file. Should I remove that indentation or just remove it from the new `@to_additive`s?

Reply:
Uh, yes, please remove the indentation everywhere, that's a porting mistake